src/main/java/com/passpnu/passwordmanager/mapper/ServiceMapper.java

```diff
@@ -0,0 +1,28 @@
+package com.passpnu.passwordmanager.mapper;
+
+import com.passpnu.passwordmanager.dto.ServiceDto;
+import com.passpnu.passwordmanager.entity.ServiceEntity;
+import org.mapstruct.Mapper;
+import org.springframework.stereotype.Component;
+
+@Mapper
```

Comment:
First of all, mapstruct, should handle these conversion automatically, you don't need to write mappings manually.
Second of all, Your mapper should be an interface with two abstract method like you already have got

Reply:
Fixed